### PR TITLE
`Fusion 360` -> `Fusion`

### PR DESCRIPTION
**Pull request by BrandonPacewic**
It's no longer called `Fusion 360` anymore...

### CODE REVIEW

**Comment by HunterBarclay** [comment]
@BrandonPacewic Can this be reviewed yet?

**Comment by BrandonPacewic** [comment]
> @BrandonPacewic Can this be reviewed yet?

~Technically it's blocked by #981. I should have specified.~

File: exporter/SynthesisFusionAddin/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Synthesis Exporter
-This is a Addin for *Autodesk Fusion 360* that will export a [Mirabuf](https://github.com/HiceS/mirabuf) usable by the Synthesis simulator.
+This is a Addin for *Autodesk Fusion* that will export a [Mirabuf](https://github.com/HiceS/mirabuf) usable by the Synthesis simulator.
```
Comment (HunterBarclay) [inline exporter/SynthesisFusionAddin/README.md]:
```suggestion
This is a Addin for Autodesk® Fusion™ that will export a [Mirabuf](https://github.com/HiceS/mirabuf) usable by the Synthesis simulator.
```

File: exporter/SynthesisFusionAddin/src/Parser/SynthesisParser/RigidGroup.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ def ExportRigidGroups(
     - Appears to have a bug, logged already
 
     Args:
-        fus_occ (adsk.fusion.Occurrence): Fusion 360 Occurrence Reference
+        fus_occ (adsk.fusion.Occurrence): Fusion Occurrence Reference
         hel_occ (Assembly_pb2.Occurrence): Protobuf Hellion Occurrence Reference
```
Comment (HunterBarclay) [inline exporter/SynthesisFusionAddin/src/Parser/SynthesisParser/RigidGroup.py:32]: Lmao, Hellion. I think that was Shawns project in like 2020-21

**Review by HunterBarclay — CHANGES_REQUESTED** [review]
Just a quick change

File: exporter/SynthesisFusionAddin/src/Parser/SynthesisParser/RigidGroup.py (hunk 2 above)
Comment (BrandonPacewic) [reply exporter/SynthesisFusionAddin/src/Parser/SynthesisParser/RigidGroup.py:32]: Ancient.

**Review by LucaHaverty — APPROVED** [review]
Looks good! I can finally sleep at night knowing we don't have "Fusion 360" in our source code.

**Review by HunterBarclay — APPROVED** [review]
👍

**Pull request merged**